coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Language and docstyle independent documentation comments extraction.
+
+Each of the functions is built upon one another, and at the last,
+exposes a single function py:function:`extract_documentation_with_markers`
```

Comment:
Sorry, I don't quite get what you mean. I didn't add a reference to function? I added py:function:`extract_documentation_with_markers`, and as far as I understood from [this](http://www.sphinx-doc.org/en/stable/markup/inline.html):
> Basically, you only need to write :role:`target`, and a link will be created to the item named target of the type indicated by role.

The so [answer](https://stackoverflow.com/questions/7753805/sphinx-automodule-how-to-reference-classes-in-same-module/7754150#7754150) does not seem to add anything but this :class: or :therm: prefix to respective entity names.